digitaltwins/samples/device/javascript/sample_device.js

```diff
@@ -28,10 +28,10 @@ const commandHandler = (request, response) => {
 };
 
 
-const environmentalSensor = new EnvironmentalSensor('environmentalSensor', propertyUpdateHandler, commandHandler);
+const environmentalSensor = new EnvironmentalSensor('sensor', propertyUpdateHandler, commandHandler);
 const deviceInformation = new DeviceInformation('deviceInformation');
 
-const capabilityModel = 'dtmi:azureiot:samplemodel;1';
+const capabilityModel = 'dtmi:YOUR_COMPANY_NAME_HERE:sample_device;1';
```

Comment:
@anthonyvercolano and I prefer the updated `contoso_device_corp` to `YOUR_COMPANY_NAME_HERE`

Reply:
Can you sync with the C-SDK samples? I'm just trying to be consistent with the samples

Reply:
+1